SpatialGDK/Source/SpatialGDKServices/Private/SpatialCommandUtils.cpp

```diff
@@ -449,6 +380,38 @@ void SpatialCommandUtils::TryKillProcessWithName(const FString& ProcessName)
 	}
 }
 
+void SpatialCommandUtils::TryGracefullyKill(const FString& ProcName, const FProcHandle& ProcHandle)
+{
+#if !PLATFORM_WINDOWS
+	// On UNIX this sends a SIGTERM signal
+	FGenericPlatformProcess::TerminateProc(ProcHandle);
+#else
+	// TerminateProc is too forceful on Windows
+	TryGracefullyKillWindows(ProcName);
+#endif
```

Comment:
very minor: generally its considered good practice to do positive case first, reads a bit easier.
```suggestion
#if PLATFORM_WINDOWS
	// TerminateProc is too forceful on Windows
	TryGracefullyKillWindows(ProcName);
#else
	// On UNIX this sends a SIGTERM signal
	// TODO: does this work on Mac?
	FGenericPlatformProcess::TerminateProc(ProcHandle);
#endif
```

Reply:
yeah that is a fair point

Reply:
Nice catch with the `FGenericPlatformProcess`! I think I agree with you on the positive case being first as well, will commit the suggestion. Thanks!